Add format flag for pkgbase_id

```
aur.c:347:21: warning: ‘aur_pkg_get_pkgbase_id’ defined but not used [-Wunused-function]
 static unsigned int aur_pkg_get_pkgbase_id (const aurpkg_t *pkg)
```

--- a/doc/package-query.8
+++ b/doc/package-query.8
@@ -247,6 +247,7 @@ Format can contain:
 %G: git clone url
 %i: AUR ID
 %I: install script
+%k: base package ID (AUR)
 %l: local version
 %L: last submission (AUR)
 %m: maintainer or packager
@@ -266,7 +267,7 @@ Format can contain:
 %U: software url
 %v: version, depends on DB
 %V: package version in sync db
-%w: AUR votes
+%w: votes (AUR)
 .fi
 .if n \{\
 .RE

--- a/src/aur.c
+++ b/src/aur.c
@@ -911,6 +911,10 @@ const char *aur_get_str (void *p, unsigned char c)
 			info = itostr (aur_pkg_get_id (pkg));
 			free_info = 1;
 			break;
+		case 'k':
+			info = itostr (aur_pkg_get_pkgbase_id (pkg));
+			free_info = 1;
+			break;
 		case 'm':
 			info = (char *) aur_pkg_get_maintainer (pkg);
 			break;

